Create ReactNativeTouchThroughView.podspec

RichardLindhout opened the pull request.

RichardLindhout committed:
Create ReactNativeTouchThroughView.podspec

martsie commented:
Thanks for your PR. Could I please get a little more information about why it's important to have a podspec. Does react-native link not work with Cocoa Pods enabled projects?

RichardLindhout commented:
I switched to podspec a while ago because of this library: https://github.com/tiero/react-native-360, but maybe it's better to make linking working in that library. Removed the react-native-360 a while ago so I'm not using podspecs anymore.
